README.md

```diff
@@ -138,6 +138,34 @@ Options:
   }
   ```
 
+### As a PostCSS Plugin
+
+UnCSS can be used as a [PostCSS](https://github.com/postcss/postcss) Plugin.
+
+```js
+postcss([ require('postcss-uncss').postcssPlugin ]);
+```
+
+See [PostCSS docs](https://github.com/postcss/postcss) for examples for your environment.
+
+**Note:** Depending on your environment, you might not be able to use giakki/uncss as a PostCSS plugin since the plugin is not directly exported. In such cases, use the wrapper library [postcss-uncss](https://github.com/RyanZim/postcss-uncss).
+
+#### Options
```

Comment:
Ah hmm, fair point. (Sorry, been too long since I've worked with this.) Maybe choose one just to demonstrate the example, and then let the reader translate it into their build tool of choice?